### DISCUSSION TITLE
Get a look to appsheet.com

### DISCUSSION THREAD

**Discussion by jechaviz** [directus/directus#2887] [Feature Requests]
Take a look of www.appsheet.com google app and www.integromat.com. These have a bunch of ideas for your app. Take what you can. 

Thank you.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
